content/docs/command-reference/remote/add.md

```diff
@@ -367,6 +367,22 @@ $ dvc remote add -d myremote \
 
 <details>
 
+### Click for OSF
+
+An [Open Science Framework](https://osf.io/) account is needed to use
+OSF storage. Please also run `pip install osfclient` to install the Python
```

Comment:
Is there good link to point "OSF storage" to? I looked in osf.io and cos.io and the best I could find is https://help.osf.io/hc/en-us/articles/360054528874 which isn't that great (doesn't explain what OSF storage is or how it works, etc.)

Reply:
Per https://github.com/iterative/dvc/pull/4860#pullrequestreview-536210649 this paragraph doesn't seem to be accurate, actually?

Reply:
I agree that https://help.osf.io/hc/en-us/articles/360054528874 fits best here.